components/ILIAS/Mail/classes/class.ilMailTemplateContextService.php

```diff
@@ -130,8 +130,7 @@ protected static function getContextInstance(
         if (!$a_path) {
             $a_path = $a_component . '/classes/';
         }
-        $class_file = $a_path . 'class.' . $a_class . '.php';
-
+        $class_file = $_SERVER['DOCUMENT_ROOT'] . $a_path . 'class.' . $a_class . '.php';
```

Comment:
This looks dangerous to me. Maybe we should skip the `file_exists` check and assume that `class_exists` does the trick (by relying that all ILIAS files are part of the "Composer Class Map").

Reply:
To be save I would also recommend to skip "file_exists" as the class always existed in the class map in my tests and the correct mail was sent. 

File_exists does  have some issues - I read somewhere that it won't work all the time - especially if you have an http adress (e.g. when a developer has a local machine running without SSL).

Would you be okay with me removing 'file_exists'?

Reply:
Yes, you could remove this.